src/test/java/uk/gov/hmcts/reform/et/syaapi/controllers/DocumentGenerationControllerTest.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.reform.et.syaapi.controllers;
+
+import lombok.SneakyThrows;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import uk.gov.hmcts.reform.et.syaapi.service.VerifyTokenService;
+import uk.gov.hmcts.reform.et.syaapi.utils.ResourceUtil;
+
+import java.io.IOException;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.reform.et.syaapi.utils.TestConstants.TEST_SERVICE_AUTH_TOKEN;
+
+@WebMvcTest(
+    controllers = {DocumentGenerationController.class}
+)
+@Import(DocumentGenerationController.class)
+
+class DocumentGenerationControllerTest {
+
+    private final String requestJson = ResourceUtil.resourceAsString(
+        "requests/caseId.json"
+    );
```

Comment:
To avoid obscure errors/exceptions when resource not found, better to initialise this variable in either a constructor, or in an @Setup method